.github/workflows/main.yml

```diff
@@ -50,6 +50,19 @@ jobs:
     - run: npm ci --prefix tests/zkasm
     - run: ./ci/test-zkasm.sh
 
+  spectest_zkasm:
```

Comment:
I suggest we merge this with the previous `test_zkasm` job:
-  They are semantically in the same category
-  We will likely have more of these tests (for each of spec test files) and we wouldn't want a new job for each one of them
- This will make sure we reuse NPM cache between jobs and save execution time as that is the dominating step for this job

Am I missing any reasons to split these into separate jobs?